test/buf.gen.yaml

```diff
@@ -1,8 +1,9 @@
 version: v2
 plugins:
-  - remote: buf.build/protocolbuffers/python:v32.0
+  # NOTE: v26.0 is the earliest version supporting protobuf==5.
```

Comment:
IIRC, this causes a warning when running with the latest protobuf, which would happen on CI for `highest`, and eventually it might fail completely if such compatibility ends. 

This is probably fine, especially since I can't think of anyway to improve it... Maybe we should copy the above content into the comment?